src/status_im/contexts/wallet/common/collectibles_tab/view.cljs

```diff
@@ -5,32 +5,34 @@
     [react-native.core :as rn]
     [status-im.common.resources :as resources]
     [status-im.contexts.wallet.common.empty-tab.view :as empty-tab]
-    [utils.i18n :as i18n]
-    [utils.re-frame :as rf]))
+    [utils.i18n :as i18n]))
 
 (defn- view-internal
-  [{:keys [theme]}]
-  (let [specific-address (rf/sub [:wallet/current-viewing-account-address])
-        collectible-list (if specific-address
-                           (rf/sub [:wallet/collectibles-per-account specific-address])
-                           (rf/sub [:wallet/all-collectibles]))]
-    (if (empty? collectible-list)
+  [{:keys [theme collectibles filtered? on-collectible-press]}]
+  (let [no-results-match-query? (and filtered? (empty? collectibles))]
+    (cond
+      no-results-match-query?
+      [rn/view {:style {:flex 1 :justify-content :center}}
+       [quo/empty-state
+        {:title       (i18n/label :t/nothing-found)
+         :description (i18n/label :t/try-to-search-something-else)
+         :image       (resources/get-themed-image :no-collectibles theme)}]]
+
+      (empty? collectibles)
       [empty-tab/view
        {:title       (i18n/label :t/no-collectibles)
         :description (i18n/label :t/no-collectibles-description)
         :image       (resources/get-themed-image :no-collectibles theme)}]
+
```

Comment:
here also NL

Reply:
These new lines are helpful when using a `cond` that has its "then" clausules beind the conditions. It's a common pattern in Clojure code